### PR TITLE
Created rss feed for blog

### LINKED ISSUE #2491: [Blog] RSS Feed: Create feed for all posts
**Current Behavior**
Site visitors can browse [blog posts](https://layer5.io/blog), but not subscribe for notifications of newly published posts.

**Desired Behavior**
Create `/blog/feed.xml` - an RSS feed containing all posts.

---
**Screenshots / Mockups**
Screen Shot 2022-01-16 at 9 03 49 PM

**Implementation**
1. Create new RSS feed in `gatsby-config.js` to be available at `/blog/feed.xml`.
1. Add RSS icon with link to feed adjacent to /blog page title as pictured in mockup.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by subhradwip01**
**Description**

This PR fixes #2491

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -165,6 +165,72 @@ module.exports = {
         ],
       },
     },
+    {
+      resolve: "gatsby-plugin-feed-mdx",
```
Comment (YashKamboj) [inline gatsby-config.js:169]: As you can see in my pr , we agreed on adding all the feed in one query, can check the comments in my pr .

**Review by YashKamboj — CHANGES_REQUESTED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (subhradwip01) [reply gatsby-config.js:169]: Ok..understood...so let me modify it according to your pr and then add my blog feed

Comment (subhradwip01) [reply gatsby-config.js]: Can you check it, please?

Comment (YashKamboj) [reply gatsby-config.js]: @subhradwip01 Yeah looks good now

**Review by YashKamboj — APPROVED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (subhradwip01) [reply gatsby-config.js]: Thanks..😀

**Comment by leecalcote** [comment]
it's great to see this. A build and preview deployment are being created now...

**Comment by l5io** [comment]
🚀 Preview for commit f218fa969593826a6b6055f2a789a79d072e3d21 at: https://61e8b59125f7ef5a9902de99--layer5ng.netlify.app

File: gatsby-config.js [hunk 2]
```diff
@@ -162,6 +143,53 @@ module.exports = {
...
+                    fields {
+                      collection
+                      slug
+                    }
+                  }
+                }
+              }
+            `,
+            output: "/blog/feed.xml",
+            title: "Layer5"
```
Comment (Nikhil-Ladha) [inline gatsby-config.js:191]: Make it `Layer5 Blogs`

**Review by Nikhil-Ladha — COMMENTED** [review]
There needs to be another update which is to merge all the feeds into one, but since it has been already done in the other PR, I am merging that....please fetch and rebase and make the update accordingly.

File: gatsby-config.js (hunk 2 above)
Comment (subhradwip01) [reply gatsby-config.js:191]: I also merge the all feeds in one..so should I do it again after fetching new update?

Comment (subhradwip01) [reply gatsby-config.js:191]: So do I need to to create a new branch for that?

Comment (subhradwip01) [reply gatsby-config.js:191]: As I am seeing there is conficits

**Review by Nikhil-Ladha — APPROVED** [review]

File: gatsby-config.js (hunk 2 above)
Comment (subhradwip01) [reply gatsby-config.js]: I was just started to do that...almost on the way of completion...But thanks..😅

Comment (Nikhil-Ladha) [reply gatsby-config.js]: Ahhh....extremely sorry for the inconvenience 😬

**Comment by l5io** [comment]
🚀 Preview for commit 0eaf29dbac5e30b48c0e6420c50bf56188120966 at: https://61e98a13b5a501416264a64e--layer5ng.netlify.app

File: gatsby-config.js (hunk 2 above)
Comment (subhradwip01) [reply gatsby-config.js]: 😂😂..Not a problem....by the way, you like a superhero for helping people

**Comment by Nikhil-Ladha** [comment]
Can you just make one quick change?
And that is to make the link open in a new tab.

**Comment by Nikhil-Ladha** [comment]
Just add the `target=_blank` to the `a` tag in the oage header component.

**Comment by subhradwip01** [comment]
@Nikhil-Ladha I mak the changes...but I forget to sign-off..Sorry for that...is it possible to sign-off that commit..or I need to mak a new commit

**Comment by l5io** [comment]
🚀 Preview for commit e959db80c7880222d8035458cc1fc1e9f916fc3e at: https://61e9a17465746c5c90bf5063--layer5ng.netlify.app

**Comment by YashKamboj** [comment]
@subhradwip01 when you click on details next dco https://github.com/layer5io/layer5/pull/2514/checks?check_run_id=4886150615

**Comment by subhradwip01** [comment]
@YashKamboj @Nikhil-Ladha can you please check everything is ok or not?

**Comment by l5io** [comment]
🚀 Preview for commit b0791fe816c9936be9bbe4de18ecf877a37a21ce at: https://61e9b561fa13b677db11a8be--layer5ng.netlify.app

**Comment by Abhijay007** [comment]
@subhradwip01 the blog RSS feed is not available when we switch between the different views of the blogs. The RSS feed is not available for the list view state, working fine for grid view.

https://user-images.githubusercontent.com/64387054/150433340-ffede90f-f35c-4161-aaf2-af28fa36f150.mp4

**Review by Abhijay007 — CHANGES_REQUESTED** [review]
@subhradwip01 the blog RSS feed is not available when we switch between the different views of the blogs. The RSS feed is not available for the list view state, working fine for grid view.

https://user-images.githubusercontent.com/64387054/150433340-ffede90f-f35c-4161-aaf2-af28fa36f150.mp4

**Comment by subhradwip01** [comment]
@Abhijay007 ok understood..I am doing it now

**Comment by l5io** [comment]
🚀 Preview for commit ee887a966e62d2ae685f128b693fe2284fc0cd20 at: https://61ea34c64b66f06fbec9efcb--layer5ng.netlify.app

**Pull request merged**